src/material/stepper/_stepper-theme.scss

```diff
@@ -15,6 +16,25 @@
   $warn: map.get($config, warn);
 
   .mat-step-header {
+    @include a11y.high-contrast(active, off) {
+      margin-bottom: 1px;
+      margin-top: 1px;
+
+      &:not([aria-disabled]),
+      &[aria-disabled='false'] {
+        outline: solid 1px;
+        &.cdk-keyboard-focused, &.cdk-program-focused {
+          outline: solid 3px;
+        }
+      }
+
+      &[aria-selected='false'] {
+        .mat-step-icon, .mat-step-label {
```

Comment:
Wouldn't this make the icons harder to see? We could potential use `text-decoration: underline` to distinguish the selected step.

Reply:
Yeah the icons are harder to see, although adding the `text-decoration: undeline` looks good to me so I changed it.